Dockerfile

```diff
@@ -38,6 +49,16 @@ RUN conda install --yes r-irkernel r-plyr r-devtools r-rcurl r-dplyr r-ggplot2 r
 RUN julia -e 'Pkg.add("IJulia")'
 RUN julia -e 'Pkg.add("Gadfly")' && julia -e 'Pkg.add("RDatasets")'
 
+# IHaskell
+ENV PATH /home/jovyan/.cabal/bin:$PATH
+RUN cabal update && \
+    cabal install cpphs && \
+    cabal install gtk2hs-buildtools && \
+    cd && git clone --depth 1 https://github.com/gibiansky/IHaskell.git && \
```

Comment:
@gibiansky Does IHaskell make releases? I'd rather pin it down to at least a tag if I can to prevent breaking our entire build or the experience users would see.